src/ToggleSwitch.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react';
+import classNames from 'classnames';
+import './ToggleSwitch.scss';
+
+type ToggleSwitchProps = {
+    ariaLabel: string,
+    value: boolean,
+    className?: string,
+    contentsTrue: any,
+    contentsFalse: any,
+    onChange: (value: boolean) => void
+};
+
+export default class ToggleSwitch extends React.Component<ToggleSwitchProps, {}> {
+    handleClick = () => {
+        this.toggleStateChange();
+    }
+
+    handleKeyDown = (e: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        const spaceKey = ' ';
+        if (e.key === spaceKey) {
+            e.preventDefault();
+            this.toggleStateChange();
+        }
+    }
+
+    toggleStateChange() {
+        this.props.onChange(!this.props.value);
+    }
+
+    render() {
+        const classes = classNames(
+            this.props.className,
+            'ToggleSwitch'
+        );
+        return (
+            <div
+                className= {
+                    this.props.value ?
+                    `${classes} ToggleSwitch--checked` :
+                    classes
+                }
```

Comment:
We can include the `ToggleSwitch--checked` class name in the `classNames` call above:

    const classes = classNames(
        this.props.className,
        'ToggleSwitch',
        { 'ToggleSwitch--checked': this.props.value }
    );

And then in the JSX, `className` becomes just: `className={classes}`